NitroxClient/GameLogic/Entities.cs

```diff
@@ -122,6 +128,11 @@ private IEntitySpawner ResolveEntitySpawner(Entity entity)
 
             TechType techType = entity.TechType.Enum();
 
+            if (entity.ClassId == "55d7ab35-de97-4d95-af6c-ac8d03bb54ca")
+            {
+                return new CellRootSpawner();
+            }
+
             if (customSpawnersByTechType.ContainsKey(techType))
             {
                 return customSpawnersByTechType[techType];
```

Comment:
Would it make sense to make this customSpawnersByClassId?  Then we can put those hard coded class ids into an enum so it would be more readable.

Reply:
while we can do that it also adds more complexity